src/pages/login/login.scss

```diff
@@ -0,0 +1,18 @@
+html {
```

Comment:
Such a global style would be better to move higher (app.scss or kind of)

Reply:
selector removed
https://github.com/mtalstykh-exadel/team4_frontend/blob/27f79d12b2ded8e5b31bd2b44a4a2d0ffd8adfa8/src/pages/login.scss#L1-L8